go.mod

```diff
@@ -1,29 +1,27 @@
 module github.com/spacemeshos/go-spacemesh
 
 require (
-	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/VividCortex/gohistogram v1.0.0 // indirect
-	github.com/btcsuite/btcd v0.0.0-20181130015935-7d2daa5bfef2
-	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
+	github.com/btcsuite/btcd v0.0.0-20190213025234-306aecffea32
```

Comment:
have a quick look at  these dependencies, just make sure no redundant dependancies were added

Reply:
Since this dependency was already existing, let's wait with attempting to remove it.

Reply:
Since they were already existing, i'll wait with the attempt to remove them.